inc/azure_c_shared_utility/xlogging.h

```diff
@@ -39,19 +39,32 @@ typedef void(*LOGGER_LOG)(LOG_CATEGORY log_category, const char* file, const cha
 #define xlogging_set_log_function(...)
 #define LogErrorWinHTTPWithGetLastErrorAsString(...)
 #define UNUSED(x) (void)(x)
+
+#elif defined(ESP8266_RTOS)
+#include "c_types.h"
+#define LogInfo(FORMAT, ...) do {    \
+        static const char flash_str[] ICACHE_RODATA_ATTR STORE_ATTR = FORMAT;  \
+        printf(flash_str, ##__VA_ARGS__);   \
+        printf("\n");\
+    } while(0)
+
+#define LogError LogInfo
+#define LOG(...)
```

Comment:
Please consider to add the code to print logs from the LOG call. It can ignore the log_category and log_options, and act as the LogInfo.